src/main/java/net/fabricmc/loader/transformer/FabricTransformer.java

```diff
@@ -45,6 +72,8 @@
 	}
 
 	public static byte[] transform(boolean isDevelopment, EnvType envType, String name, byte[] bytes) {
+		bytes = fixLegacyGuava(bytes);
```

Comment:
Why transform Guava if they're on the modern versions?

Reply:
because it wont affect new versions